models.rb

```diff
@@ -36,7 +36,7 @@ class MenuItem
 	property :description,			Text, required: true
 	
 	has n, :reviews
-	has n, :lunch_menu_items
+	has n, :lunch_menu_items, constraint: :destroy
```

Comment:
Yes, this is the best.  I found an example on [this StackOverflow thread](http://stackoverflow.com/questions/14740195/destroying-dependents-in-datamapper).